[2022-08-03 02:20:10,292] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T02:10:00+00:00 [queued]>
[2022-08-03 02:20:10,325] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T02:10:00+00:00 [queued]>
[2022-08-03 02:20:10,326] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 02:20:10,327] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-03 02:20:10,327] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 02:20:10,343] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-03T02:10:00+00:00
[2022-08-03 02:20:10,349] {standard_task_runner.py:51} INFO - Started process 30117 to run task
[2022-08-03 02:20:10,355] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'respond', '2022-08-03T02:10:00+00:00', '--job-id', '516', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmpg3y4m01v']
[2022-08-03 02:20:10,359] {standard_task_runner.py:76} INFO - Job 516: Subtask respond
[2022-08-03 02:20:10,433] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.respond 2022-08-03T02:10:00+00:00 [running]> on host b4dcda255553
[2022-08-03 02:20:10,520] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T02:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-03T02:10:00+00:00
[2022-08-03 02:20:10,521] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-03 02:20:10,560] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=respond, execution_date=20220803T021000, start_date=20220803T022010, end_date=20220803T022010
[2022-08-03 02:20:10,601] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-03 02:20:10,646] {local_task_job.py:118} INFO - Task exited with return code 0
